finalPrep/modZeroHero.js

```diff
@@ -1,41 +1,61 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName;
+var specialAbility;
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeting = 'Have no fear, ' + heroName + ' is here!';
+var catchphrase = `I just so happen to have the ability to ${specialAbility}`;
 
 // Declare two variables - power AND energy - set to integers
+var power = 9000;
+var energy = 173;
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
-
+var fullPower = power * 500;
+var fullEnergy = energy + 150
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = false;
+var identityConcealed = true;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ['Bad Horse', 'Tacos R. Gonezo', 'Silent Butt-Deadly'];
+var sidekicks = ['Left Man', 'Hero Girl', 'Amazing Cat Face'];
 // Print the first sidekick to your console
-
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2]);
```

Comment:
Would there be a way to address the prompt from line 33 and the code work for an array of ANY length??

Reply:
Updated to 'console.log(archEnemies[archEnemies.length -1]);'